lms/static/sass/multicourse/_home.scss

```diff
@@ -19,30 +24,34 @@
     padding: 0;
     width: flex-grid(12);
 
-    .outer-wrapper {
+    .outer-wrapper, .course-search {
+      @include clearfix();
       @extend .animation-home-header-pop-up;
-      margin: 0 auto;
-      padding: 240px ($baseline/2) 0;
       position: relative;
-      max-width: 1200px;
-      min-width: 760px;
-      @include clearfix();
+      margin: 0 auto ($baseline);
+      padding: 0 ($baseline/2);
+      min-width: ($baseline*38);
+      max-width: ($baseline*60);
+    }
+
+    .outer-wrapper {
+      padding-top: ($baseline*5);
     }
 
     .title {
-      background: $white;
-      background: $course-header-bg;
-      border: 1px solid $border-color-3;
-      box-shadow: 0 4px 25px 0 rgba(0,0,0, 0.5);
+      @include margin-left($title-left-margin);
+      @include float(left);
       @include box-sizing(border-box);
-      min-height: 120px;
-      margin-left: grid-width(2) + $gw-gutter;
-      width: flex-grid(6);
-      float: left;
-      position: relative;
       @include transition(all 0.2s linear 0s);
+      position: relative;
+      border: 1px solid $border-color-3;
+      box-shadow: 0 4px 25px 0 rgba(0,0,0, 0.5);
```

Comment:
When using RGB notation, always add a single space after a comma (,) and no space between parentheses ((, )) and content. ~styleguide